app/services/session.js

```diff
@@ -70,6 +70,7 @@ export default class Auth0Service extends Service {
 
     let result = await this.auth0.handleRedirectCallback();
     this.isAuthenticated = await this.auth0.isAuthenticated();
+    // QUESTION: does this work on Safari? We've had issues in the past using `?` in Safari
```

Comment:
This will work without any problem in `Safari` for sure. And I had never heard of such a problem before—tho here is some console test.
![Screen Shot 2022-08-14 at 7 12 22 PM](https://user-images.githubusercontent.com/19944631/184547662-d726e8d0-4e98-4bae-8a61-8826b7522364.png)

Reply:
Possibly due to the version of Babel we are using in our older Ember app